### PR TITLE
Add nvm for linux

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
- Add nvm to ubuntu

Initial issue: https://github.com/actions/virtual-environments/issues/4

### CODE REVIEW

**Comment by maxim-lobanov** [comment]
Could you please update title to reflect OS where you install NVM?
Could you please update description with the list of changes and mention initial issue?

File: images/linux/scripts/installers/nvm.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+export NVM_DIR="/opt/.nvm"
+mkdir -m 777 $NVM_DIR
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+chmod -R 777 $NVM_DIR
+echo "NVM_DIR=$NVM_DIR" | tee -a /etc/environment
+echo '[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm' >> /etc/profile.d/nvm.sh
+echo '[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completio' >> /etc/profile.d/nvm.sh
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/nvm.sh]: typo: `bash_completio`

File: images/linux/scripts/installers/nvm.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+################################################################################
+##  File:  nvm.sh
+##  Desc:  Installs Nvm
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+export NVM_DIR="/opt/.nvm"
+mkdir -m 777 $NVM_DIR
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/nvm.sh]: Why we need this magic with permissions in two places? 11 and 13 lines.
If they are necessary, please put small comment

File: images/linux/scripts/installers/nvm.sh (hunk 1 above)
Comment (dmitry-shibanov) [reply images/linux/scripts/installers/nvm.sh]: Done.

File: images/linux/scripts/installers/nvm.sh (hunk 2 above)
Comment (dmitry-shibanov) [reply images/linux/scripts/installers/nvm.sh]: Done.

File: images/linux/scripts/installers/nvm.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+################################################################################
+##  File:  nvm.sh
+##  Desc:  Installs Nvm
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+export NVM_DIR="/etc/skel/.nvm"
+# should create directory with full permissions.
```
Comment (sergey-akhalkov) [inline images/linux/scripts/installers/nvm.sh:11]: It is still not clear why we need full permissions here and below, could you please clarify and add a comment?

Comment (dmitry-shibanov) [reply images/linux/scripts/installers/nvm.sh]: Yes, It was previously added when nvm was installed to /opt/cause it's system dir. Nvm installs new node versions to its dir. But now we install It to skel and nvm dir copies to $HOME dir of user, so I removed this action. I'll rerun my checks if everything is ok I'll leave a comment and close conversation.

Comment (dmitry-shibanov) [reply images/linux/scripts/installers/nvm.sh]: @sergey-akhalkov , removed full permissions and work as expected

**Comment by boredland** [comment]
Hi! How is this supposed to work? I tried this with this miniature workflow and it didn't:
```
name: nvm
on:
  push:
jobs:
  nvm:
    runs-on: ubuntu-latest
    steps:
      - name: try out nvm
        run: nvm install
```

**Comment by al-cheb** [comment]
@boredland , please don't forget shell params:
`shell: bash -l {0}`

 -l        Make bash act as if it had been invoked as a login shell (see INVOCATION below).
       --login
              files ~/.bash_profile, ~/.bash_login, or ~/.profile.  By default, bash reads these  files  when  it  is
              invoked as a login shell (see INVOCATION below).

**Pull request merged**